web3.js/src/transaction.ts

```diff
@@ -17,6 +17,22 @@ import type {CompiledInstruction} from './message';
  */
 export type TransactionSignature = string;
 
+export type TransactionBlockhash = {
+  blockhash?: Blockhash;
+  lastValidBlockHeight?: number;
```

Comment:
Why optional? Is it not true that, under the new config regime, people _have_ to bring this information in order to be able to use the new implementation of `confirmTransaction`?

Reply:
I initially tried to keep things similar to other methods on the `Transaction` class where `recentBlockhash` is marked as optional. However, I agree that with the new config options being implemented it doesn't make sense to make these optional. I'll fix this when I refactor it.